### ISSUE TITLE
Disable rating after it is rated

### ISSUE THREAD

**Issue by asifsami73** [callmenick/five-star-rating#12]
I wanna disable the hover and rating effet after the user rate so that he will be unabe to rate again pls somebody help

**Comment by raualvron** [comment]
I was wondering if it is possible as well. Did you get a solution @asifsami73?

**Comment by asifsami73** [comment]
yep i have got the solution you need to add disabled with the class name
